src/mustache_core.js

```diff
@@ -225,7 +225,15 @@ var core = {
 							return localPartialName(partialScope, {}, nodeList);
 						} else {
 							var domRenderer = core.getTemplateById(localPartialName);
-							return domRenderer ? domRenderer(partialScope, {}, nodeList) : getDocument().createDocumentFragment();
+							if (domRenderer) {
+								return domRenderer(partialScope, {}, nodeList);
+							}
+
+							//!steal-remove-start
+							if (process.env.NODE_ENV !== 'production') {
+								dev.warn('Unable to find the partial "' + localPartialName + '" in ' + state.filename + ':' + state.lineNo);
```

Comment:
I think you are missing a `steal-remove-end`.

Also, I think this code would make more sense if you keep the `return` code how it was and make the warning separate.

```js
// !steal-remove-start
if (process.env.NODE_ENV !== 'production') {
  if (!domRenderer){
    ...
  }
}
// !steal-remove-end
```

retrun domRenderer ? ...;

Reply:
Also, can you add the filename and lineNumber?

Reply:
It was returned before I changed, no? `domRenderer ? domRenderer(partialScope, {}, nodeList) : getDocument().createDocumentFragment()`

Reply:
@phillipskevin fixed!